### PR TITLE
Configured the search integration able to query sellers registered

### PR DESCRIPTION
**Pull request by Frankdevlopes**
integrated the search bar to the backend  enabling easier searching of  sellers and products.

### CODE REVIEW

File: src/components/shared/SearchBar/SearchBar.tsx [hunk 1]
```diff
@@ -1,109 +1,167 @@
...
 
 interface searchBarProps {
-  onSearch?:(query:string) => void;
-  page: 'map_center' | 'default'; 
+  onSearch?: (query: string) => void;
+  page: 'map_center' | 'default';
 }
 
 const SearchBar: React.FC<searchBarProps> = ({ onSearch, page }) => {
-  const t = useTranslations();
```
Comment (swoocn) [inline src/components/shared/SearchBar/SearchBar.tsx:22]: Not sure why the translation was removed..?

**Comment by swoocn** [comment]
Hi @Frankdevlopes - Good effort!
I’ve reviewed and tested your search integration changes and noticed some flaws with it.  Though you followed the search implementation from the Angular version, some parts won’t work as expected due to the lazy loading capabilities in the current app.  Instead of zooming out when a search query is submitted, we need to stay on the same viewport and display only the map markers that match the search criteria.  

I'll be making some adjustments to this PR, along with backend changes to leverage the existing fetchSellers API.  I figure we can simply incorporate the search query as an additional parameter with the coordinates.  I'll keep you posted so you can review the updates and understand the modifications.  Thanks!

**Review by swoocn — CHANGES_REQUESTED** [review]
I will be making the adjustments so @Frankdevlopes can continue to proceed with the other tasks.

**Comment by Frankdevlopes** [comment]
Thanks danny i will be waiting for the changes that you will implement i think that is a good approach to incorporate the search query as additional parameter. Eager to see the modification in the mean time i will continue with the recent assigned task.

**Pull request merged**